spec.md

```diff
@@ -209,6 +209,23 @@ that contains both context and data).
     [RFC 2046](https://tools.ietf.org/html/rfc2046)
 * For Media Type examples see [IANA Media Types](http://www.iana.org/assignments/media-types/media-types.xhtml)
 
+### property-bucket
+* Type: <Key Value>
+* Description: A list of “key : value” pairs in flat strcuture that specify the properties of the event source.
```

Comment:
`strcuture` -> `structure`

Reply:
Is it worth making clear that the properties are specific to the source, `specify the properties of the event source` could be interpreted as properties that apply to multiple sources.

Reply:
That might be a thing. In the motion sensors example that sends an event to an apartment-level gateway and then a building-level gateway there would indeed be properties that apply to multiple events( say address). See https://github.com/cloudevents/spec/pull/128#issuecomment-395239721 for some context and examples.

Reply:
Will fix the typo. 
Vlad, Not sure what you mean by multiple events. I would view your example as the "same" event from the eventual event consumer's point of view (e.g. the serverless platform) , with intermediate GWs adding more info or modifying the original event info. As to whether the source is the original sender or the last mile GW, that is a separate discussion.

Reply:
@cathyhongzhang I deleted a follow-up comment asking if was worth clarifying whether properties were source specific or common to all sources. Deleted it as I'd raised it during the call without digesting @Vlaaaaaaad's detailed summary.